### DISCUSSION TITLE
Midi Implementation Chart

### DISCUSSION THREAD

**Discussion by JoaquinSainz** [probonopd/MiniDexed#221] [Q&A] [answered]
Hey how's everybody doing?
Is there a Midi Implementation Chart for minidexed?

**Comment by probonopd** [comment, marked as answer]
Hi @JoaquinSainz. @diyelectromusic and @miotislucifugis have started to make one:
Please have a lok at https://github.com/probonopd/MiniDexed/wiki/MIDI-Implementation.

**Reply by JoaquinSainz** [reply]
Thanks👍
